src/Starcounter.VS.CSBoilerplateApplicationProjectTemplate/Api/MainHandlers.cs

```diff
@@ -0,0 +1,31 @@
+﻿using $safeprojectname$.ViewModels;
+using Starcounter;
+
+namespace $safeprojectname$.Api
+{
+    internal static class MainHandlers
+    {
+        public static void Register()
+        {
+            Application.Current.Use(new HtmlFromJsonProvider());
+            Application.Current.Use(new PartialToStandaloneHtmlProvider());
+
+            Handle.GET("/$safeprojectname$", () =>
+            {
+                var page = new HelloStarcounterPage
+                {
+                    Visitors = {Data = Db.SQL<GuestBookEntry>("SELECT g FROM GuestBookEntry g")}
+                };
+
+                if (Session.Current == null)
+                {
+                    Session.Current = new Session(SessionOptions.PatchVersioning);
+                }
+
+                page.Session = Session.Current;
+
+                return page;
```

Comment:
I suppose that ASP.NET MVC has a checkbox to say if you want just boilerplate or also sample code.

Also, why keep it different than our HelloWorld app that is located here https://github.com/StarcounterApps/HelloWorld and explained at docs.starcounter.io? Is HelloWorld too big for this?